### DISCUSSION TITLE
Multilingual associations

### DISCUSSION THREAD

**Discussion by JurgenG** [joomla/joomla-cms#47195] [2. Cold Features]
### Steps to reproduce the issue
Have a multilingual site where categories are linked between languages
(e.g. category "dogs" is linked with French category "chiens")
Open an article in the category dogs and create a French target article.
Copy over the content

### Expected result
The target category should be the French category associated with the English category
Possibly a check could be done if the language for that category is set to the target language first and/or the reference category language is not "all".

### Actual result
The article defaults in the same category as the reference (it just copied over the values)

### System information (as much as possible)
Joomla 3.7.0.rc1

### Additional comments
I don't know if this is really a bug, but I think it would be a really smart improvement.

**Comment by infograf768** [comment]
Copy Reference to Target does what it says: it copies all from reference to target.
One has to edit the newly created target to set the category, usually also the title and alias, and the contents.

Not a bug

**Comment by alikon** [comment]
is not a bug agree, but can be an improvement/new feature if the reference and target category are associated already

**Comment by brianteeman** [comment]
I agree it would be a logical​ I improvement

**Comment by gwsdesk** [comment]
Sorry but it is a bug me think. Copy Reference to Target is not working and bugs out as shown in the screen-cast several times. [https://www.screencast.com/t/aiQj8TGG9Ym](https://www.screencast.com/t/aiQj8TGG9Ym "Screencast") Sometimes it works though so kind of weird. Happens both in FF and Chrome and 1 bug as well namely if you change the target language from Dutch to English the "copy Reference" button has disappeared. This is probably since English is set as main language for the site since changing to Dutch or to French it does not disappear. I am not quite sure why it disappears only with a change to Target language selector English [https://www.screencast.com/t/pWfjrId0](https://www.screencast.com/t/pWfjrId0 "Disappearing Target") <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15206">issues.joomla.org/tracker/joomla-cms/15206</a>.</sub>

**Comment by infograf768** [comment]
not at my desk. will check your screencast tomorrow

**Comment by infograf768** [comment]
I can't reproduce.
![multiassoc](https://cloud.githubusercontent.com/assets/869724/25268981/13aca45c-267b-11e7-942a-34400f926993.gif)

**Comment by infograf768** [comment]
+ your issue is totally unrelated to site or admin default languages

**Comment by JurgenG** [comment]
Additional to the first suggestion (automatically switch category of content to associated catagory in target language), you could do the same to menu items in a menu. Here it is even more critical I think because when trying to create a language frontpage, it's not even allowed to be in the same category (one can only have one primary page per menu)

**Comment by JurgenG** [comment]
@infograf768, if your last comment targeted to the poster of the screencast or to my original request? I agree with you this isn't a real bug. I do think in terms of user experience, it's confusing. So while it technically isn't a bug, you could see it as a UX bug. (just like "drive straight forward" also means you have to follow the curves of the road)

**Comment by infograf768** [comment]
@JurgenG 
The javascript using the 'simple' existing `save2copy` method is already extremely complex.
https://github.com/joomla/joomla-cms/blob/staging/media/com_associations/js/sidebyside-uncompressed.js
Although these improvements would be great to get in, we would need a very serious coder to work on such aspects.
Volunteers welcome.

**Comment by JurgenG** [comment]
@infograf768, here I stand by your side. Improvements are only as feasible as the people who can (skill/time) implement them. So a big congrats to this great feature. I yesterday demonstrated multilingual sites on our JUG and people were impressed about how great this improvement is.

**Comment by rdeutz** [comment]
Would be an improvement for sure, the question is: when? The usual situation is that we make a copy within the same category/menu. So when should the system behave different, is it when we have a multi language site? If it is only in that specific view or situation, then we are coupling extensions together and this will limit us in the future. 

To keep a long story short, this is by far not an easy task if we will do it right.

**Comment by brianteeman** [comment]
@coolcat-creations This looks like it is being addressed in the GSOC project ??

**Comment by coolcat-creations** [comment]
no not really @brianteeman

[a comment by an automated account is not shown]

**Comment by jwaisner** [comment]
Closing as features are not available for J3. Please review J4 and open a feature request if this is still a wanted feature.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15206">issues.joomla.org/tracker/joomla-cms/15206</a>.</sub>

**Comment by brianteeman** [comment]
this should not be closed!!!

it is a perfectly valid feature request and should be considered for joomla 4
